src/discovery/azure/Akka.Discovery.Azure/AkkaHostingExtensions.cs

```diff
@@ -123,11 +128,17 @@ public static class AkkaHostingExtensions
             TableClientOptions? tableClientOptions = null,
             string? serviceName = null,
             string? publicHostname = null,
-            int? publicPort = null)
+            int? publicPort = null,
+            string discoveryId = "azure",
```

Comment:
What does `discoveryId` mean?

Reply:
Discovery Id is the HOCON configuration key the final HOCON settings will live in. For example, "azure" the default Id will be placed in "akka.discovery.azure", just like how Akka.Persistence journal Id works. This will allow it to be addressed and loaded by the Akka.Discovery plugin later on.